### DISCUSSION TITLE
Merging PR and fixing Issues

### DISCUSSION THREAD

**Discussion by LDprg** [winapps-org/winapps#12] [General]
I think we could start merging PRs and fixing Issues. However how do we merge stuff from the old repo? Do we transfer the PRs and review stuff or do we just do a direct commit?

**Comment by oskardotglobal** [comment]
Since I've kinda rewritten history we would probably have to manually apply the patch manually since the 2 repos trees are diverted

**Reply by oskardotglobal** [reply]
Also, we shouldn't merge anything without having a CLA signed by the author first

**Reply by LDprg** [reply]
Yeah, of course. I will focus on the rewrite since it isn't affected of the problem at all.
